style.css

```diff
@@ -1,340 +1,1403 @@
+    gap: 24px;
+}
+
+.blog__list,
+.blog__blockquote,
+.intro__subtitle,
+.blog-content__photo {
+    margin-bottom: 32px;
+}
+
+.blog__list--mark {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    list-style-type: none;
+}
+
+.blog__list {
+    padding-left: 20px;
+    list-style-type: inherit;
+}
+
+.blog__list--mark {
+    padding-left: 0;
+}
+
+.blog__list--mark  li {
+    position: relative;
+    padding-left: 30px;
+    margin-bottom: 8px;
+    list-style-type: none;
+}
+
+.blog__list--mark li:before {
+    position: absolute;
+    content: '';
+    background-image: url('images/list-icon.svg');
+    width: 24px;
+    height: 24px;
+    left: 0;
+}
+
+.blog__text--link {
+    color: #78BBFA;
+}
+
+.blockquote__wrapper {
+    margin-bottom: 30px;
+    padding: 16px;
+    background-color: #002754;
+    border-radius: 16px;
+}
+
+.blockquote {
+    font-size: 24px;
+    font-weight: 600;
+    line-height: 30px;
+}
+
+.blockquote__rating {
+    position: relative;
+    padding-left: 30px;
+    margin-bottom: 0;
+}
+
+.blockquote__rating:before {
+    position: absolute;
+    content: '';
+    background-image: url('images/stars.svg');
+    width: 16px;
+    height: 16px;
+    top: 2px;
+    left: 0;
+}
+
+.blockquote__span {
+    padding-right: 10px;
+    opacity: 0.8;
+}
+
+.blockquote p {
+    margin-bottom: 0;
+}
+
+cite {
+    font-style: normal;
+    font-weight: 600;
+}
+
+.blog__photo-caption {
+    margin-top: -30px;
+    margin-left: 20px;
+}
+
+.sidebar__title {
+    display: block;
+    margin-bottom: 16px;
+    padding-bottom: 16px;
+    font-family: var(--font-secondary);
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 32px;
+    border-bottom: 1px solid rgba(191, 197, 238, 0.12);
+}
+
+.categories__item {
+    padding: 6px 12px;
+}
+
+.tags__wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 8px;
+}
+
+.tags__item {
+    padding: 6px 12px;
+    font-size: 14px;
+    line-height: 20px;
+    border: 2px solid rgba(208, 215, 251, 0.17);
+}
+
+.blog-footer {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    column-gap: 30px;
+    font-size: 16px;
+    line-height: 24px;
+}
+
+.blog-footer__wrapper {
+    display: flex;
+    align-items: center;
+    column-gap: 20px;
+}
+
+.subtitle--center {
+    text-align: left;
+    margin-bottom: 16px;
+}
+
+.blog-content .btn.btn--color {
+    margin-bottom: 30px;
+}
+
+/*Contact page */
+
+
+.contact-info {
+    display: flex;
+    justify-content: center;
+    column-gap: 16px;
+    margin-bottom: 65px;
+}
+
+.contact-info__item {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    min-width: 328px;
+    padding: 40px;
+    border: 1px solid #D4D9F242;
+}
+
+.contact-info__item img {
+    margin-bottom: 24px;
+}
+  
+.input-hidden {
+    position: absolute;
+    width: 0;
+    height: 0;
+    overflow: hidden;
+    pointer-events: 0;
```

Comment:
Incorrect property. Should be pointer-events: none;

Reply:
oops. thank you. Done